manuscript/040_Three_Essential_Tools.md

```diff
@@ -295,7 +296,7 @@ public class ExplodingOrderDatabase : OrderDatabase
 }
 ```
 
-Ok, so far so good, but the bad thing is that now we have got two classes of fake objects to maintain (and chances are we will need even more). Any method or argument added will need to be propagated to all these objects. We can spare some coding by making our mocks a little more generic so their behavior can be configured using lambda expressions:
+Ok, so far so good, but now we have two classes of fake objects to create (and chances are we will need even more). Any method or argument added will need to be propagated to all these objects. We can spare some coding by making our mocks a little more generic so that their behavior can be configured using lambda expressions:
```

Comment:
No problem. "Propagate" is probably not the most used word anyway and "Add" in this context is clear enough.